rpp30-boc-meltdown/biteshare
# Summary screen final

## Description
This PR is a combined work of @tm2838 and me. It contains refactoring in the Summary Screen logic and QR Code logic. After this PR is merged, when the Guest will scan the QR code, he will be added to the database and when the HOST will allow him to join, he will be redirected to the Summary (or do we need to redirect the guest to Menu page?) Page. This PR also contains some CSS stylings refactoring.

## How to test
Steps to test this:
1. Have two different devices.
2. From one device, you create a session as a HOST.
3. From the other device, you scan the QR code as a GUEST.
4. HOST will see who has scanned the code in the Summary Screen
5. GUEST will be redirected to Summary Screen when the host allows him.

## Notes


## Issue tracking
https://trello.com/c/jXZXeOgf
